charters/DeiCharter.md

```diff
@@ -0,0 +1,31 @@
+# Diversity Equity and Inclusion Committee Charter
+
+Project Jupyter welcomes and encourages participation in our community by people of all backgrounds and identities. We are committed to promoting and sustaining a culture that values mutual respect, tolerance, and learning. We work together as a community to help each other live out these values.
+
+The Diversity Equity & Inclusion (DEI) Committee is a standing committee established under Jupyter’s governance model. The committee reviews its charter annually and may make changes at any time during the year. Substantive changes to the purpose or mission must be submitted to the Executive Council (EC) and the Software Steering Council (SSC) for approval.
+
+## Purpose 
+
+The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, contributors, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. The DEI group impacts all Jupyter forums, including but not limited to software, working groups, standing committees, JupyterCon, social media, GitHub, the EC, and the SSC.
+
+## Mission/Areas of Responsibility
+
+The DEI standing committee is responsible for: 
+* Propose, execute, and evaluate the impact of DEI initiatives
+* Research, establish, and collect metrics related to DEI
+* Advocate for increased participation of underrepresented groups in the Jupyter community. 
+* Serve as a point of contact for DEI topics for the global Jupyter community
+* Include the topic of DEI in discussions and spaces within Project Jupyter.
+* Recommend and promote training, educational, and motivational materials on DEI 
+* Flag DEI issues or potential issues to the EC or the SSC
+
+## Meetings
+* Frequency - this committee should meet at least once every two weeks.
+* The Jupyter Decision-Making Guide will be used to make official decisions. 
+* To ensure active participation, a member who is unable to join for more than two thirds of regularly scheduled weekly meetings in a 90 day period may be asked to step down from the committee.
```

Comment:
Thx @Zsailer for asking about the sync/async as timezone. Sorry to hijack this PR, but would like to mention we had the exact same discussions during the last Software Steering Council (SSC). Should those aspects be defined for all groups, or are they a delegated to the groups? In the latter case the groups would be free to decide and evolve the rules by themselves and those aspects should not be mentioned in the charter IMHO.

Thinking loud, I would prefer the governance to define some guidance for all groups to have a basis and common framework to avoid discussion and too much disparity across groups.

BTW the timezone question is a perfect topic for the DEI group to ensure enough on-call representation from around the globe :)

Reply:
The delay linked to vote mentioned in the Decision-Making Guide is a year. Is there a specific reason to have 90 days here?

> Voter participation and quorum. All members of a council are required to participate in at least 2/3 of formal votes of that council per calendar year (councils can decide how to account for the specifics of this in low-activity projects, etc.). Members that have not met the 2/3 vote participation threshold for a year will automatically be asked to step down at the end of that year.

Reply:
@Zsailer Thanks for bringing this up. The "time zone problem", as I'll call it for short, affects our team and many others. On the DEI committee, we have made an effort to set weekly meetings at a time that is convenient for most of our members. We currently have four members on the DEI committee, all of whom are in the Americas, and we are focused on synchronous participation. We are definitely open to holding our meetings at different times if we find someone for whom our current meeting time is not suitable. We use the word "may" in our charter to say that it is not automatic to exclude a person for being unable to attend 2/3 of the meetings. Other committees should set expectations for themselves. Does this address your concern?

@fcollonval Our charter describes an expectation (but not a requirement) to attend more than 2/3 of scheduled weekly meetings. The decision-making guide describes a requirement to participate in at least 2/3 of formal votes. Because a scheduled weekly meeting is not as formal, we don't consider our guidance to be in conflict with the decision-making guide.